framework/src/executioners/FixedPointSolve.C

```diff
@@ -552,3 +559,12 @@ FixedPointSolve::autoAdvance() const
 
   return auto_advance;
 }
+
+bool
+FixedPointSolve::performingRelaxation(const bool primary) const
+{
+  if (primary)
+    return _relax_factor != 1.;
+  else
+    return _secondary_relaxation_factor != 1.;
```

Comment:
I always pause at floating point comparisons. This is probably fine. I'll leave it to you whether to keep as-is or do a fuzzy equal comparison